packages/synapse-react-client/src/components/SynapseTable/usePrefetchTableData.ts

```diff
@@ -78,15 +81,15 @@ function usePrefetchFileHandleData() {
   )
 }
 
-function usePrefetchEntityData() {
+function useGetEntitiesInTable() {
```

Comment:
Extracted hook to get all of the entities shown in the table. We were just using this to get entity data, but now we can use it to get entity data and restriction information